DOCS/man/input.rst

```diff
@@ -2930,6 +2930,11 @@ Property list
     version >= 4 is used (LVDS-1, HDMI-A-1, X11-1, etc.), or the wl_output model
     reported by the geometry event if protocol version < 4 is used.
 
+    ``display-names/friendly``
+        These are the human-friendly names obtained from the monitor EDID
+        (Dell AW3223DW, LC34G55T, etc.). These follow the same order as
+        ``display-names`` (Windows only)
```

Comment:
Might it be worth making the property available on all systems, but merely act as a duplicate of `display-names` for non-windows systems? That way scripts, keybinds, and auto-profiles don't need to worry about the platform when grabbing useful display names, they could just grab `display-names/friendly` and have it work on all systems.

This presumably wouldn't prevent friendlier (EDID) names from being added for those those backends at a later date if people wanted.

Reply:
On linux, the display names are already friendly (i.e. like `DP-1`). I think macOS is similar. As opposed to "friendly" names, this seems to be more like the model name of the display. It's certainly possible to add it to other platforms.

Reply:
We can handle this new VOCTRL everywhere, for now with the same impl, and extended if needed.

Reply:
just as info, on macOS it's something like `DELL U2723QE (12343224)`, name + serial number.